app/services/PaypalService.scala

```diff
@@ -59,27 +87,69 @@ class PaypalService(config: PaypalApiConfig) {
       val links = createdPayment.getLinks.asScala
       val approvalLink = links.find(_.getRel.equalsIgnoreCase("approval_url"))
       approvalLink.map(l => Right(l.getHref)).getOrElse(Left("No approval link returned from paypal"))
-    }
-    catch {
+    } catch {
       case e: PayPalRESTException => Left(e.getMessage)
     }
   }
 
-  def executePayment(paymentId: String, token: String, payerId: String): Either[String, Unit] = {
+  def executePayment(paymentId: String, payerId: String): Either[String, String] = {
```

Comment:
Yeah I was bugged by that `Either[String, Unit]` even though I doubt it's actually wrong